src/Microsoft.Identity.Web.TokenAcquisition/MicrosoftIdentityBaseAuthenticationBuilder.cs

```diff
@@ -40,18 +40,23 @@ public class MicrosoftIdentityBaseAuthenticationBuilder
 
         internal static void SetIdentityModelLogger(IServiceProvider serviceProvider)
         {
+            var config = serviceProvider.GetRequiredService<IConfiguration>();
+            var loglevel = config.GetValue<string>("Logging:LogLevel:Microsoft.Identity.Web");
+            
             if (serviceProvider != null)
```

Comment:
no need to check for serviceProvider here because it will throw on line 43 if it's not found.

Reply:
Good point. I'll move these lines inside that block so that it doesn't throw in that case